oracle_cardano/processor/tx_processors/success/bridging_requested_processor_skyline.go

```diff
@@ -0,0 +1,345 @@
+			totalAmountWrappedTokenSrc += uint64(0)
+
+			totalAmountDest += receiver.Additional.DestinationCurrencyAmount
+			totalAmountWrappedTokenDest += receiver.Amount
+		}
+
+		receivers = append(receivers, cCore.BridgingRequestReceiver{
+			DestinationAddress: receiverAddr,
+			Amount:             new(big.Int).SetUint64(receiverAmount),
+			AmountWrapped:      new(big.Int).SetUint64(receiverAmountWrapped),
+		})
+	}
+
+	feeAmount := metadata.FeeAmount.DestinationCurrencyAmount
+
+	totalAmountDest += feeAmount
+
+	receivers = append(receivers, cCore.BridgingRequestReceiver{
+		DestinationAddress: feeAddress,
+		Amount:             new(big.Int).SetUint64(feeAmount),
+		AmountWrapped:      new(big.Int).SetUint64(0),
+	})
+
+	claim := cCore.BridgingRequestClaim{
+		ObservedTransactionHash:         tx.Hash,
+		SourceChainId:                   common.ToNumChainID(tx.OriginChainID),
+		DestinationChainId:              common.ToNumChainID(metadata.DestinationChainID),
+		Receivers:                       receivers,
+		NativeCurrencyAmountDestination: new(big.Int).SetUint64(totalAmountDest),
+		NativeCurrencyAmountSource:      new(big.Int).SetUint64(totalAmountSrc),
+		WrappedTokenAmountSource:        new(big.Int).SetUint64(totalAmountWrappedTokenSrc),
+		WrappedTokenAmountDestination:   new(big.Int).SetUint64(totalAmountWrappedTokenDest),
+		RetryCounter:                    big.NewInt(int64(tx.BatchFailedCount)),
+	}
+
+	claims.BridgingRequestClaims = append(claims.BridgingRequestClaims, claim)
+
+	p.logger.Info("Added BridgingRequestClaim",
+		"txHash", tx.Hash, "metadata", metadata, "claim", cCore.BridgingRequestClaimString(claim))
+}
+
+/*
+func (*BridgingRequestedProcessorSkylineImpl) addRefundRequestClaim(
+	claims *core.BridgeClaims, tx *core.CardanoTx, metadata *common.BridgingRequestMetadata,
+) {
+
+		var outputUtxos []core.Utxo
+		for _, output := range tx.Outputs {
+			outputUtxos = append(outputUtxos, core.Utxo{
+				Address: output.Address,
+				Amount:  output.Amount,
+			})
+		}
+
+		// what goes into UtxoTransaction
+		claim := core.RefundRequestClaim{
+			TxHash:             tx.Hash,
+			RetryCounter:       0,
+			RefundToAddress:    metadata.SenderAddr,
+			DestinationChainId: metadata.DestinationChainId,
+			OutputUtxos:        outputUtxos,
+			UtxoTransaction:    core.UtxoTransaction{},
+		}
+
+		claims.RefundRequest = append(claims.RefundRequest, claim)
+
+		p.logger.Info("Added RefundRequestClaim",
+		"txHash", tx.Hash, "metadata", metadata, "claim", core.RefundRequestClaimString(claim))
+}
+*/
+
+func (p *BridgingRequestedProcessorSkylineImpl) validate(
+	tx *core.CardanoTx, metadata *common.BridgingRequestMetadata, appConfig *cCore.AppConfig,
+) error {
+	chainConfig := appConfig.CardanoChains[tx.OriginChainID]
+	if chainConfig == nil {
+		return fmt.Errorf("unsupported chain id found in tx. chain id: %v", tx.OriginChainID)
+	}
+
+	if err := utils.ValidateOutputsHaveTokens(tx, appConfig); err != nil {
```

Comment:
this is only for reactor.. it errors out if there are native tokens in utxo to bridging addr.. for skyline, we should support that utxos have native tokens